fuzzers/fuzzbench_forkserver/src/main.rs

```diff
@@ -372,6 +373,7 @@ fn fuzz(
             .debug_child(debug_child)
             .shmem_provider(&mut shmem_provider)
             .parse_afl_cmdline(&arguments)
+            .is_persistent(true)
```

Comment:
With the latest changes in AFL++, this flag is no longer needed.
That being said, the delayed forkserver still is.

Reply:
still better to have ti in case of an old afl-cc